template/content/.gitignore

```diff
@@ -13,10 +13,10 @@ output
 *.cache
 
 # IDEs
-/.idea/*.iml
-/.idea/*.xml
-/.idea/.idea.*/*.iml
-/.idea/.idea.*/.idea/*.xml
+/**/.idea/*.iml
+/**/.idea/*.xml
+/**/.idea/.idea.*/*.iml
+/**/.idea/.idea.*/.idea/*.xml
```

Comment:
I'm not 100% sure, but I think you may omit the `/**/` part.

`.idea/*.xml` should do the same. Could you please check?

Reply:
This was my initial idea, but seems like `.idea/*.xml` works only for `.idea` folder on the same level as `.gitignore`

Reply:
🤯